src/RBACHandler.test.ts

```diff
@@ -68,7 +73,7 @@ const noGroupsDecoded = {
 };
 const nonPractitionerDecoded = {
     sub: 'fake',
-    'cognito:groups': ['non-practitioner', 'auditor'],
+    'cognito:groups': ['auditor', 'non-practitioner'],
```

Comment:
why was ordered change? if no reason, leave as original, so that we have less changes